server/src/main/java/org/elasticsearch/rest/action/admin/indices/RestPutIndexTemplateAction.java

```diff
@@ -57,6 +59,9 @@ public String getName() {
     @Override
     public RestChannelConsumer prepareRequest(final RestRequest request, final NodeClient client) throws IOException {
         PutIndexTemplateRequest putRequest = new PutIndexTemplateRequest(request.param("name"));
+        if (request.hasParam(INCLUDE_TYPE_NAME_PARAMETER)) {
+            deprecationLogger.deprecatedAndMaybeLog("put_index_template_include_type_name", TYPES_DEPRECATION_MESSAGE);
```

Comment:
Super small comment: for consistency this should be "put_index_template_with_types".